src/app/profile/layout.tsx

```diff
@@ -19,7 +19,9 @@ export default async function ProfileLayout({
       <Header nav={HEADER_NAV} user={user} />
       <main className="h-screen">
         <div className="flex mt-4 w-full h-full">
-          <ProfileSideNav />
+          <div className="ProfileSideNav">
```

Comment:
Why do we need to add this `div`? This is not a trick question, so actually curious what's the need for it.